css/index.css

```diff
@@ -23,6 +28,7 @@ h1 {
     font-size: 4rem;
     line-height: 1.5;
     margin-bottom: 10px;
+    color: #191919;
```

Comment:
If you're going to soften the black, you need to file an issue in the Concierge repo so we can make the same change for consistency.